pkg/question/question_test.go

```diff
@@ -343,6 +343,7 @@ func TestAskImage_Success(t *testing.T) {
 	const expectedImage = "ami-12345"
 	const testInstanceType = ec2.InstanceTypeT2Micro
 
+	testEC2 = ec2helper.New(session.Must(session.NewSession()))
```

Comment:
I don't see any additional tests here? are you planning to include additional tests for new methods that you added? I suggest adding tests here for new methods.

Reply:
In new revision, we modify `TestGetLatestImages` method in e2e_ec2helper_test to make sure we can get images from all different os (even without SSM path or result imageIds got from SSM path API are empty).

Also, add a new `TestGetImageIdsFromSSM` method in e2e_ec2helper_test to protect and detect do we have correct paths.